src/invert.ts

```diff
@@ -0,0 +1,46 @@
+import { purry } from './purry';
+
+type Invertable = Record<PropertyKey, PropertyKey> | Array<PropertyKey>;
+type Inverted = Record<PropertyKey, PropertyKey>;
```

Comment:
Here is what I came up with for typing

https://www.typescriptlang.org/play?#code/PTAEDUEMDsEsBt6VAFwJ4AdbQOYCg90MBTUASWgDdiAnFYgEwB4AVUYgD3ugYGdQA9gCMAVsQDGKAHygAvKBYBtANbE0AgGYKAuuy7Ee-AAo0BJOmgDSavKFAB+UACUJAmsyWr1WltoA0oF6aClK2oABcoNDE1DQA3AREpACCcuRUtPTMAN6gkJEAjHGgQpEATKAAvlIJSaAAQmkUsVlMLuJuzAAGABbEiAIA+gAk2bwoNNg4lV0BXQDubvAMI2MTUzNSNYSYpADCTRl0jEy5+SUCAvDEMFXbeCCgAOqwKD2ouwC0GsTjibugPYCHivWDAyDwaxoXhMeqQXjEAJAkEoMHQGTyABywMxAFdEJAhNcmGFsmE7IooaBsIE1ME4QjtJEGcRKWpdJxuHxAcCGKDgQ5QFTItFYgk7JUVHStCztHhtnVkXzUeD4EZYOJlLD4YieSi0RjQOrNSS7Cy-GElfzoBCoTDzXrlQb5QkdiR0i1GC83ixdgAxX4oVh6Ln8YRiSSGzzSnQhgzckxmTJWGx2Ry5RRGanQBRS7w6Jm0-NWlU2tUarUsAJGGSVMIimK0BJAA

Reply:
TypeFest is really neat! I wish I would have know about that sooner.